_labo-help/similarity.markdown

```diff
@@ -24,52 +26,51 @@ The Similarity tool offers the following features and functionality.
+The keyframes are shown in a grid under the media viewer. They provide a visual and auditory overview of the current video. When the video is playing, the currently active keyframe is highlighted. Clicking on a tile moves the player video position to the keyframe timestamp. This way one can navigate the video, in addition to the left/right buttons that move playback to the previous or next keyframe.
 
-The keyframes grid is also available in the resource viewer. It provides the same control about the media playback as in the similarity tool.
+The keyframes grid is also available in the resource viewer. It provides the same control for the media playback as in the similarity tool.
 
 ### <a name="related-keyframes"></a>Related keyframes
 
-Related keyframes are loaded for the active keyframe. Results can be controlled using the query options. Use the info icon behind each input to learn more about its use!
+Related keyframes are loaded for the active keyframe. Results can be controlled using the query options. Use the info icon behind each input to learn more about its use. 
 
-Based on the input keyframe, similar keyframes will be found by comparing the distance between visual and auditory feature vectors that have been created using the [VisXP model](#technology). The vector search itself is handled by an Elastic Search backend. Search scores can be displayed by using the score button:
+Based on the active keyframe in the left panel, similar keyframes are retrieved by comparing the distance between visual and auditory feature vectors that have been created using the [VisXP model](#technology). Search scores can be displayed by using the score button:
 
 ![Similarity tool score button](/uploads/similarity-tool-score.jpg)
 
-In the related keyframes grid overview, the resources can be bookmarked to the project and annotations can be added to the specific segments:
+### <a name="annotation"></a>Annotation and bookmarking
 
-![Similarity tool annotation](/uploads/similarity-tool-annotation.jpg)
+Using the annotation button, annotations can be made on a segment that spans the start and end time of the detected shot that the keyframe belongs to. This applies to both the [current resource's keyframes](#keyframes) and the [related keyframes](#related-keyframes)
 
-### <a name="annotation"></a>Annotation
+![Similarity tool annotation](/uploads/similarity-tool-annotation.jpg)
 
 Annotations you make will show up in the Resource Viewer, which you can conveniently open by using the resource viewer button on the top right.
 
 Annotations are always added to a new layer called 'Similarity'. Feel free to edit the contents in the resource viewer. However, keep in mind that: 1) the Similarity tool will always create a new layer called Similarity if not present, and 2) if you change the start/end time of segments, the similarity tool can't use them for bookmarking anymore.
```

Comment:
the similarity tool can't use them for bookmarking anymore.

ik snap niet precies wat dit betekent. Bookmarken is anders dan annoteren. Verdwijnen de bookmarks als je de start/end tijd veranderd?

Reply:
Goeie vraag, dit komt uit Werners help-tekst dus ik zal hem even om uitleg vragen

Reply:
Fixed now